packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -1497,6 +1498,67 @@ module.exports = function (log, error) {
       })
   }
 
+  Memory.prototype.createAccountSubscription = async function (uid, subscriptionId, productName, createdAt) {
+    // Ensure user account exists
+    uid = uid.toString('hex')
+    await getAccountByUid(uid)
+
+    const key = [uid, subscriptionId, productName].join('|')
+    if (accountSubscriptions[key]) {
+      return P.reject(error.duplicate())
```

Comment:
Now it's switched over to `async`, could also just `throw` these errors insted of `return P.reject(...)`

Reply:
Oh yeah, I did that in some places but missed this one!